index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="welcome-section">
+      <h1>My portfolio</h1>
+    </section>
+
+    <section id="projects">
+      <h1 class="project-tile">My project</h1>
+      <div>
+        <a id="profile-link" href="https://github.com/wassimchakib/html_css_dami_wassim"
+      target = "_blank"> 
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" >
```

Comment:
Add `alt` attribute to all your images. This is used as an alternative if by any chance the image cannot be found or displayed. check the sample below.

`<img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png"  alt="image description">`